environment.yml

```
@@ -66,7 +66,7 @@ dependencies:
   - scipy>=1.10
   - statsmodels==0.13.5
   - sympy~=1.12
-  - xarray>=2024.05.0
+  - xarray>=2025.01.0
```

Actually, might as well update to the latest version released a couple of days ago - https://github.com/pydata/xarray/releases/tag/v2025.03.0

```suggestion
  - xarray>=2025.013.0
```